src/FluentEmailProvider/EmailService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using FluentEmail.Core;
+using RazorLight;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace FluentEmailProvider
+{
+    public class EmailService : IEmailService
+    {
+
+        private IFluentEmail _fluentEmail;
+
+        public EmailService(IFluentEmail fluentEmail)
+        {
+            _fluentEmail = fluentEmail;
+        }
+
+        public async void SendEmail(string email, string templatePath, string message, object model)
```

Comment:
string message parameter is never used, since it's part of the model anyways, it should be removed.

Reply:
removed